db/migrate/20170717140915_add_numeric_columns_to_container_quota_items.rb

```diff
@@ -0,0 +1,44 @@
+class AddNumericColumnsToContainerQuotaItems < ActiveRecord::Migration[5.0]
+  class ContainerQuotaItem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    add_column :container_quota_items, :quota_desired_float,  :float
+    add_column :container_quota_items, :quota_desired_int,    :bigint
```

Comment:
Why have both an int *and* a float?

Reply:
> P.S. this may be out of scope, but I wish we stored units somewhere, and not just inferred them from resource

In most every other column where we care about this we've chosen a base unit, so there is no need for storing a unit.  For example, for disk sizes we stores bytes always, and if the code receives something else it just converts to bytes.  This is very important for presentation purposes, because they can be assured that whatever they get back from the database is the same unit, and then they can apply consistent formatting rules during presentation (e.g. showing as megabytes or whatever).  I recommend doing the same here.

Reply:
IIUC in this table we have mixed *quantities* which can't have one unit: a quota where `resource=="cpu"` is in cores, a quota where `resource=="memory"` (?) is in bytes...

Right, it'd be wasteful to actually *store* a unit in every row.  I just think it's a question the *model* should be able answer (based on the `resource`), rather than having this logic elsewhere.